addons/bestja_project/views/project.xml

```diff
@@ -30,7 +30,10 @@
                     <sheet>
                         <div class="oe_right one_button_width oe_button_box" name="buttons">
                             <button class="oe_inline oe_stat_button" type="action" name="%(action_project_message)d" icon="fa-envelope" groups="bestja_project.managers">
-                                <div>Wiadomość</div>
+                                <div>Wyślij</div>
+                            </button>
+                            <button class="oe_inline oe_stat_button" type="action" name="%(mail.action_mail_archives_feeds)d" icon="fa-inbox" groups="bestja_project.managers" context="{'search_default_model': 'bestja.project','search_default_res_id': active_id}">
```

Comment:
I don't think you need to restrict this to project managers. Project members also can use this to view mail their received from this project.